### PR TITLE
[WIP] Add check for empty rootfs

### PR DESCRIPTION
**Pull request by jashook**
Make sure that the generated rootfs is created correctly.

### CODE REVIEW

File: src/ubuntu/build-scripts/build-rootfs.sh [hunk 1]
```diff
@@ -28,4 +28,12 @@ do
     docker run --privileged --rm --name $buildRootFSContainer -e ROOTFS_DIR=/rootfs/$arch \
         -v $PWD/rootfs:/rootfs -v $scriptsVolume:/scripts \
         $dockerCrossDepsTag /scripts/cross/build-rootfs.sh $arch $crossToolset $lldb --skipunmount
+
+    # Check if the rootfs/<arch> exists and is not empty
+    if [ ! -d "$PWD/rootfs/arch" ]; then
```
Comment (MichaelSimons) [inline src/ubuntu/build-scripts/build-rootfs.sh]: Shouldn't `arch` be `$arch`?

Comment (jashook) [reply src/ubuntu/build-scripts/build-rootfs.sh]: yes thank you

Comment (sdmaclea) [reply src/ubuntu/build-scripts/build-rootfs.sh]: This is what I was going to add
```
    if [[ ! -d $PWD/rootfs/$arch/bin ]]; then
        echo "Rootfs build failed: missing directory rootfs/$arch/bin"
        exit 1
    fi
```

Comment (MichaelSimons) [reply src/ubuntu/build-scripts/build-rootfs.sh]: I like @sdmaclea proposal.

**Pull request closed without merging**